invoicing/models.py

```diff
@@ -0,0 +1,41 @@
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+from enumfields import EnumField
+
+from invoicing.enums import (
+    InstallmentPercentageSpecifier,
+    InstallmentType,
+    InstallmentUnit,
+)
+
+
+class InstallmentBase(models.Model):
+    type = EnumField(InstallmentType, verbose_name=_("type"), max_length=32)
+    value = models.DecimalField(
+        verbose_name=_("value"), max_digits=16, decimal_places=2
+    )
+    unit = EnumField(InstallmentUnit, verbose_name=_("unit"), max_length=32)
+    percentage_specifier = EnumField(
+        InstallmentPercentageSpecifier,
+        verbose_name=_("percentage specifier"),
+        max_length=32,
+        blank=True,
+        null=True,
+    )
+    account_number = models.CharField(max_length=255, verbose_name=_("account number"))
+    due_date = models.DateField(verbose_name=_("due date"), blank=True, null=True)
+
+    class Meta:
+        abstract = True
+
+
+class ProjectInstallment(InstallmentBase):
+    project_uuid = models.UUIDField(verbose_name=_("project UUID"))
+
+
+class ApartmentInstallment(InstallmentBase):
+    apartment_uuid = models.UUIDField(verbose_name=_("apartment UUID"))
+    remarks = models.CharField(max_length=1024, verbose_name=_("remarks"), blank=True)
+    laske_reference = models.CharField(
```

Comment:
This is missing from the Factory class. Is this something extra still which would be better to add later if we example create the invoice class or something like that?

Reply:
Added it to the factory 👍 Also renamed the field to `reference_number`, not sure why I had used such a confusing name 🙂